keyboards/melgeek/z70ultra/info.json

```diff
@@ -0,0 +1,26 @@
+        {"label":"Ctrl", "x":0, "y":4, "w":1.25}, {"label":"Win", "x":1.25, "y":4, "w":1.25}, {"label":"Alt", "x":2.5, "y":4, "w":1.25}, {"x":3.75, "y":4, "w":6.25}, {"label":"Alt", "x":10, "y":4, "w":1.25}, {"label":"Fn", "x":11.25, "y":4, "w":1.25}, {"label":"Left", "x":13, "y":4, "w":1}, {"label":"Down", "x":14, "y":4, "w":1}, {"label":"Right", "x":15, "y":4, "w":1}
+      ]
+    },
+
+    "LAYOUT_3_3_split_spacebar": {
+      "layout": [
+        {"label":"Esc", "x":0, "y":0}, {"label":"!", "x":1, "y":0}, {"label":"@", "x":2, "y":0}, {"label":"#", "x":3, "y":0}, {"label":"$", "x":4, "y":0}, {"label":"%", "x":5, "y":0}, {"label":"^", "x":6, "y":0}, {"label":"&", "x":7, "y":0}, {"label":"*", "x":8, "y":0}, {"label":"(", "x":9, "y":0}, {"label":")", "x":10, "y":0}, {"label":"_", "x":11, "y":0}, {"label":"+", "x":12, "y":0}, {"label":"Backspace", "x":13, "y":0, "w":2}, {"label":"Home", "x":15, "y":0},
+        {"label":"Tab", "x":0, "y":1, "w":1.5}, {"label":"Q", "x":1.5, "y":1}, {"label":"W", "x":2.5, "y":1}, {"label":"E", "x":3.5, "y":1}, {"label":"R", "x":4.5, "y":1}, {"label":"T", "x":5.5, "y":1}, {"label":"Y", "x":6.5, "y":1}, {"label":"U", "x":7.5, "y":1}, {"label":"I", "x":8.5, "y":1}, {"label":"O", "x":9.5, "y":1}, {"label":"P", "x":10.5, "y":1}, {"label":"{", "x":11.5, "y":1}, {"label":"}", "x":12.5, "y":1}, {"label":"|", "x":13.5, "y":1, "w":1.5}, {"label":"PgUp", "x":15, "y":1},
+        {"label":"Caps Lock", "x":0, "y":2, "w":1.75}, {"label":"A", "x":1.75, "y":2}, {"label":"S", "x":2.75, "y":2}, {"label":"D", "x":3.75, "y":2}, {"label":"F", "x":4.75, "y":2}, {"label":"G", "x":5.75, "y":2}, {"label":"H", "x":6.75, "y":2}, {"label":"J", "x":7.75, "y":2}, {"label":"K", "x":8.75, "y":2}, {"label":"L", "x":9.75, "y":2}, {"label":":", "x":10.75, "y":2}, {"label":"\"", "x":11.75, "y":2}, {"label":"Enter", "x":12.75, "y":2, "w":2.25}, {"label":"PgDn", "x":15, "y":1},
+        {"label":"Shift", "x":0, "y":3, "w":2.25}, {"label":"Z", "x":2.25, "y":3}, {"label":"X", "x":3.25, "y":3}, {"label":"C", "x":4.25, "y":3}, {"label":"V", "x":5.25, "y":3}, {"label":"B", "x":6.25, "y":3}, {"label":"N", "x":7.25, "y":3}, {"label":"M", "x":8.25, "y":3}, {"label":"<", "x":9.25, "y":3}, {"label":">", "x":10.25, "y":3}, {"label":"?", "x":11.25, "y":3}, {"label":"Shift", "x":12.25, "y":3, "w":1.75}, {"label":"Up", "x":14, "y":3},{"label":"End", "x":15, "y":3},
+        {"label":"Ctrl", "x":0, "y":4, "w":1.25}, {"label":"Win", "x":1.25, "y":4, "w":1.25}, {"label":"Alt", "x":2.5, "y":4, "w":1.25}, {"x":3.75, "y":4, "w":3}, {"x":6.75, "y":4, "w":3}, {"label":"Alt", "x":9.75, "y":4, "w":1.5}, {"label":"Fn", "x":11.25, "y":4, "w":1.25}, {"label":"Left", "x":13, "y":4, "w":1}, {"label":"Down", "x":14, "y":4, "w":1}, {"label":"Right", "x":15, "y":4, "w":1}
+    }
+  }
+}
```

Comment:
You can use a JSON linter to validate the info.json, and the QMK Configurator to preview it, by pressing Ctrl+Shift+I.

```suggestion
{
    "keyboard_name": "Z70Ultra",
    "maintainer": "MelGeek001365",
    "width": 16,
    "height": 5,
    "layouts": {
        "LAYOUT_65_ansi_blocker": {
            "layout": [
                {"x": 0, "y": 0},
                {"x": 1, "y": 0},
                {"x": 2, "y": 0},
                {"x": 3, "y": 0},
                {"x": 4, "y": 0},
                {"x": 5, "y": 0},
                {"x": 6, "y": 0},
                {"x": 7, "y": 0},
                {"x": 8, "y": 0},
                {"x": 9, "y": 0},
                {"x": 10, "y": 0},
                {"x": 11, "y": 0},
                {"x": 12, "y": 0},
                {"x": 13, "y": 0, "w": 2},
                {"x": 15, "y": 0},

                {"x": 0, "y": 1, "w": 1.5},
                {"x": 1.5, "y": 1},
                {"x": 2.5, "y": 1},
                {"x": 3.5, "y": 1},
                {"x": 4.5, "y": 1},
                {"x": 5.5, "y": 1},
                {"x": 6.5, "y": 1},
                {"x": 7.5, "y": 1},
                {"x": 8.5, "y": 1},
                {"x": 9.5, "y": 1},
                {"x": 10.5, "y": 1},
                {"x": 11.5, "y": 1},
                {"x": 12.5, "y": 1},
                {"x": 13.5, "y": 1, "w": 1.5},
                {"x": 15, "y": 1},

                {"x": 0, "y": 2, "w": 1.75},
                {"x": 1.75, "y": 2},
                {"x": 2.75, "y": 2},
                {"x": 3.75, "y": 2},
                {"x": 4.75, "y": 2},
                {"x": 5.75, "y": 2},
                {"x": 6.75, "y": 2},
                {"x": 7.75, "y": 2},
                {"x": 8.75, "y": 2},
                {"x": 9.75, "y": 2},
                {"x": 10.75, "y": 2},
                {"x": 11.75, "y": 2},
                {"x": 12.75, "y": 2, "w": 2.25},
                {"x": 15, "y": 2},

                {"x": 0, "y": 3, "w": 2.25},
                {"x": 2.25, "y": 3},
                {"x": 3.25, "y": 3},
                {"x": 4.25, "y": 3},
                {"x": 5.25, "y": 3},
                {"x": 6.25, "y": 3},
                {"x": 7.25, "y": 3},
                {"x": 8.25, "y": 3},
                {"x": 9.25, "y": 3},
                {"x": 10.25, "y": 3},
                {"x": 11.25, "y": 3},
                {"x": 12.25, "y": 3, "w": 1.75},
                {"x": 14, "y": 3},
                {"x": 15, "y": 3},

                {"x": 0, "y": 4, "w": 1.25},
                {"x": 1.25, "y": 4, "w": 1.25},
                {"x": 2.5, "y": 4, "w": 1.25},
                {"x": 3.75, "y": 4, "w": 6.25},
                {"x": 10, "y": 4, "w": 1.25},
                {"x": 11.25, "y": 4, "w": 1.25},

                {"x": 13, "y": 4},
                {"x": 14, "y": 4},
                {"x": 15, "y": 4}
            ]
        },
        "LAYOUT_65_ansi_blocker_split_space": {
            "layout": [
                {"x": 0, "y": 0},
                {"x": 1, "y": 0},
                {"x": 2, "y": 0},
                {"x": 3, "y": 0},
                {"x": 4, "y": 0},
                {"x": 5, "y": 0},
                {"x": 6, "y": 0},
                {"x": 7, "y": 0},
                {"x": 8, "y": 0},
                {"x": 9, "y": 0},
                {"x": 10, "y": 0},
                {"x": 11, "y": 0},
                {"x": 12, "y": 0},
                {"x": 13, "y": 0, "w": 2},
                {"x": 15, "y": 0},

                {"x": 0, "y": 1, "w": 1.5},
                {"x": 1.5, "y": 1},
                {"x": 2.5, "y": 1},
                {"x": 3.5, "y": 1},
                {"x": 4.5, "y": 1},
                {"x": 5.5, "y": 1},
                {"x": 6.5, "y": 1},
                {"x": 7.5, "y": 1},
                {"x": 8.5, "y": 1},
                {"x": 9.5, "y": 1},
                {"x": 10.5, "y": 1},
                {"x": 11.5, "y": 1},
                {"x": 12.5, "y": 1},
                {"x": 13.5, "y": 1, "w": 1.5},
                {"x": 15, "y": 1},

                {"x": 0, "y": 2, "w": 1.75},
                {"x": 1.75, "y": 2},
                {"x": 2.75, "y": 2},
                {"x": 3.75, "y": 2},
                {"x": 4.75, "y": 2},
                {"x": 5.75, "y": 2},
                {"x": 6.75, "y": 2},
                {"x": 7.75, "y": 2},
                {"x": 8.75, "y": 2},
                {"x": 9.75, "y": 2},
                {"x": 10.75, "y": 2},
                {"x": 11.75, "y": 2},
                {"x": 12.75, "y": 2, "w": 2.25},
                {"x": 15, "y": 2},

                {"x": 0, "y": 3, "w": 2.25},
                {"x": 2.25, "y": 3},
                {"x": 3.25, "y": 3},
                {"x": 4.25, "y": 3},
                {"x": 5.25, "y": 3},
                {"x": 6.25, "y": 3},
                {"x": 7.25, "y": 3},
                {"x": 8.25, "y": 3},
                {"x": 9.25, "y": 3},
                {"x": 10.25, "y": 3},
                {"x": 11.25, "y": 3},
                {"x": 12.25, "y": 3, "w": 1.75},
                {"x": 14, "y": 3},
                {"x": 15, "y": 3},

                {"x": 0, "y": 4, "w": 1.25},
                {"x": 1.25, "y": 4, "w": 1.25},
                {"x": 2.5, "y": 4, "w": 1.25},
                {"x": 3.75, "y": 4, "w": 3},
                {"x": 6.75, "y": 4, "w": 3},
                {"x": 9.75, "y": 4, "w": 1.5},
                {"x": 11.25, "y": 4, "w": 1.25},

                {"x": 13, "y": 4},
                {"x": 14, "y": 4},
                {"x": 15, "y": 4}
            ]
        }
    }
}
```